XCUITests/FxScreenGraph.swift

```diff
@@ -187,16 +213,26 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
 
 extension Navigator {
     // Open a URL. Will use/re-use the first BrowserTab or NewTabScreen it comes to.
-    func openURL(urlString: String) {
+    func openURL(urlString: String, regularBrowsing: Bool = true) {
```

Comment:
Ok, I left those methods as they were :) and updated the tests so they work.
We can modify the methods in the future and make a refactor of the tests.
Thanks @jhugman!